### ISSUE TITLE
The "On this page" component should be before content in DOM order

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#91]
The "On this page" component in the sidebar is currently after the document content in DOM order. Because this is a useful component for people navigating with a keyboard, it would be much better to have it be before the document content in DOM order.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563481
